core/src/main/kotlin/fr/sncf/osrd/api/api_v2/path_properties/PathPropResult.kt

```diff
@@ -0,0 +1,35 @@
+package fr.sncf.osrd.api.api_v2.path_properties
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import fr.sncf.osrd.railjson.schema.common.ID
+import fr.sncf.osrd.railjson.schema.geom.RJSLineString
+import fr.sncf.osrd.utils.units.Distance
+
+class PathPropResult(
+    val slopes: RangeValues<Double>,
+    val gradients: RangeValues<Double>,
+    val electrifications: RangeValues<Electrification>,
+    var geometry: RJSLineString
+) {
+
+    companion object {
+        val adapterResult: JsonAdapter<PathPropResult> =
```

Comment:
Same, missing distance adapter

The polymorphism on electrification requires a little extra work as well (to define which field defines the type). See [here](https://github.com/OpenRailAssociation/osrd/blob/dev/core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlockResponse.kt#L87-L96) for an example